docs/tutorial/population_dynamics.rst

```diff
@@ -151,10 +151,22 @@ physical state of the system is hidden or unobservable.
 This description is the most basic implementation of an HMM. For detailed theoretical and practical
 explanations on the use and training of HMMs see the seminal work by Rabiner :cite:`Rabiner1989HMM`.
 
-We can fit an HMM with a single line of code::
+Fit an HMM using the Baum-Welch algorithm::
 
     hmm = lk.HiddenMarkovModel(force["0s":"2s"], n_states=2)
 
+This algorithm estimates the model parameters based on the provided data.
+One of these estimated parameters is the transition matrix which can be extracted as::
+
+    >>> hmm.transition_matrix
+    array([[0.96339315, 0.03660685],
+       [0.08215035, 0.91784965]])
```

Comment:
This is very minor, but I think typically, there are a few more spaces in the repl output which help align the matrix visually.

```suggestion
    >>> hmm.transition_matrix
    array([[0.96339315, 0.03660685],
           [0.08215035, 0.91784965]])
```